external/webserver/http_client.c

```diff
@@ -476,6 +520,9 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 						http_dispatch_url(client, req);
 						process_finish = true;
 						read_finish = true;
+
+						len->sentence_start = sentence_end + 2;
+						process_trailer_header(buf, buf_len, req, params, client, len->sentence_start);
```

Comment:
it doesn't check return value from function. It can be dangerous. please handle error cases.

Reply:
done